Tests/WordPressKitTests/Tests/Utilities/URLSessionHelperTests.swift

```diff
@@ -308,28 +308,24 @@ class URLSessionHelperTests: XCTestCase {
         let _ = await session.perform(request: builder, errorType: TestError.self)
 
         // Capture a list of files in the temp dirs, after calling the upload function.
-        let tempFilesAfterUpload = existingTempFiles()
+        let tempFilesAfterUpload = try existingMultipartFormTempFiles()
 
         // There should be no new files after the HTTP request returns. This assertion relies on an implementation detail
         // where the multipart form content is put into a file in temp dirs.
         let newFiles = tempFilesAfterUpload.subtracting(tempFilesBeforeUpload)
         XCTAssertEqual(newFiles.count, 0)
     }
 
-    private func existingTempFiles() -> Set<String> {
+    // This functions finds temp files that are used for uploading multipart form.
+    // The implementation relies on an internal implementation detail of building multipart form content.
+    private func existingMultipartFormTempFiles() throws -> Set<String> {
+        let uuidRegex = try NSRegularExpression(pattern: "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-4[0-9a-fA-F]{3}-[89ab][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$")
         let fm = FileManager.default
-        let enumerators = [
-            fm.enumerator(atPath: NSTemporaryDirectory()),
-            fm.enumerator(atPath: fm.temporaryDirectory.path)
-        ].compactMap { $0 }
-
-        var result: Set<String> = []
-        for enumerator in enumerators {
-            while let file = enumerator.nextObject() as? String {
-                result.insert(file)
+        let files = try fm.contentsOfDirectory(atPath: fm.temporaryDirectory.path)
+            .filter {
+                uuidRegex.numberOfMatches(in: $0, range: NSMakeRange(0, $0.count)) == 1
```

Comment:
nitpick: this also works:
```suggestion
                UUID(uuidString: $0) != nil
```
and avoids the long regex.

Reply:
Thanks for suggestion! Addressed in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/781/commits/a7f92975139b6491519184474d7af586bec54557